public/index.html

```diff
@@ -3,24 +3,81 @@
   <head>
     <meta charset="utf-8">
     <title>Hoodie</title>
-  </head>
-  <body>
-    <h1>Hello from Hoodie!</h1>
+    <style>
+    body{
+background-color:#FCFCFC;
+animation-name: graytocolorimg;
+animation-duration: 4s;
+}
+h1{
+  font-family:fantasy,sans-serif;
```

Comment:
I think `fantasy` looks cute, but we don’t really use it anywhere else. I’d suggest to just stick with `Helvetica, Airal, sans-serif`. We can improve this later, once we put more thinking into a coherent Hoodie Design :)